src/cli/cli.ts

```diff
@@ -122,24 +122,24 @@ program
 program
     .command('install [browserType...]')
     .description('ensure browsers necessary for this version of Playwright are installed')
-    .action(async function(args) {
+    .action(async function(args: any[]) {
       try {
         // Install default browsers when invoked without arguments.
         if (!args.length) {
           await registry.installBinaries();
           return;
         }
-        const browserNames: Set<BrowserName> = new Set(args.filter((browser: any) => allBrowserNames.has(browser)));
-        const browserChannels: Set<BrowserChannel> = new Set(args.filter((browser: any) => allBrowserChannels.has(browser)));
-        const faultyArguments: string[] = args.filter((browser: any) => !browserNames.has(browser) && !browserChannels.has(browser));
+        const binaries = args.map(arg => registry.findBinary(arg)).filter(b => !!b) as RegistryBinary[];
```

Comment:
```suggestion
        const binaries = args.map(arg => registry.findBinary(arg)).filter(Boolean) as RegistryBinary[];
```
personal preference nit, does afaik the same.